VastClient/VastClient/parsers/VMAPParser.swift

```diff
@@ -40,6 +42,20 @@ class VMAPParser: NSObject {
     }
 
     func parse(url: URL) throws -> VMAPModel {
+        do {
+            if vmapArchiver.shouldUseSavedVMAP(url: url) {
```

Comment:
this doesn't necessarily meet our use case. What we are trying to acomplish is if a user has left a stream, they have 5 minutes to return to use previously saved VMAP information. So a user could have loaded a stream, watched it for 30 mintues and then the 5 minute timer begins. Not 5 minutes since we last made a vmap request.

I still ike the idea for this lib to simplify caching of the vmap model, but it might just be best for an integrator to say use cached vmap vs make a new request.